sites/partners/cypress/integration/listing.spec.ts

```diff
@@ -102,7 +102,7 @@ describe("Listing Management Tests", () => {
     cy.get("a > .button").contains("Add Listing").click()
     cy.contains("New Listing")
 
-    cy.getByID("addPhotoButton").contains("Add Photo").click()
+    cy.getByTestId("add-photos-button").contains("Add Photo").click()
```

Comment:
If we're not going to write a test here do you mind adding a ticket to do so?

Reply:
@emilyjablonski Sure, I'll see if I can expand the test there a bit.